regenmaschine/parser.py

```diff
@@ -13,3 +16,14 @@ async def current(self) -> Dict[str, Any]:
         """Get current diagnostics."""
         data = await self._request("get", "parser")
         return cast(Dict[str, Any], data["parsers"])
+
+    async def post_data(
+        self, payload: Dict[str, List[Dict[str, Any]]]
+    ) -> Dict[str, Any]:
+        """Post weather data from an external source.
+
+        Local Weather Push service should be enabled from Settings > Weather >
```

Comment:
I would point people to the entire API doc (since it will have this and other crucial info): https://rainmachine.docs.apiary.io/#reference/weather-services/parserdata/post